### PR TITLE
Commit content sample app

### PR DESCRIPTION
**Pull request by gonzalonm**

### CODE REVIEW

**Comment by gonzalonm** [comment]
Ported from https://github.com/googlesamples/android-CommitContentSampleApp

File: android-n/CommitContentSampleApp/CommitContentSampleApp/MainActivity.cs [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+			var editText = new CustomEditText(this) {MimeTypes = mimeTypes, Hint = hintText};
+			editText.SetTextColor(Color.White);
+			editText.SetHintTextColor(Color.White);
+			return editText;
+		}
+
+		private class CustomEditText : EditText
+		{
+			public string[] MimeTypes { get; set; }
+			public MainActivity Owner { get; set; }
```
Comment (pjcollins) [inline android-n/CommitContentSampleApp/CommitContentSampleApp/MainActivity.cs]: 'Owner' is never set and will throw when we try to invoke OnCommitContentListenerImpl.OnCommitContent. One solution is below:

```
public CustomEditText(MainActivity context) : base(context)
{
	Owner = context;
}
```

Also the three additional constructor declarations are not needed.

**Comment by gonzalonm** [comment]
Removed the unused constructors and using the inline assignment to set Owner object. @pjcollins what do you think about that?

**Comment by pjcollins** [comment]
Works for me, and now included in #163.

**Pull request closed without merging**